packages/forms/src/UIForm/fieldsets/Array/Array.component.test.js

```diff
@@ -94,14 +98,14 @@ describe('Array component', () => {
 				onFinish={jest.fn()}
 				schema={schema}
 				value={value}
-			/>
+			/>,
 		);
 
 		// then
 		expect(wrapper.node).toMatchSnapshot();
 	});
 
-	it('should render array that can\'t be reordered', () => {
+	it("should render array that can't be reordered", () => {
```

Comment:
why " ? it's simple quote

Reply:
Prettier do that to avoid having antislash in strings

Reply:
:+1: it's prettier, not me :)